.github/workflows/cicd.yml

```diff
@@ -37,18 +37,17 @@ jobs:
       - name: set datetime
         run: |
           echo "datetime=$(date -u +'%Y-%m-%dT%H:%M:%SZ')" >> $GITHUB_ENV
-      - name: log in registry.redhat.io
```

Comment:
I believe this should not be removed as it is required for https://github.com/juliaaano/antora-viewer/blob/e80903cd5015dfb072c42d1b0dfc051190d68381/container/Containerfile#L1

Reply:
No, registry.redhat.io need auth. 

registry.access.redhat.com works without auth and provide only "free" available images like the ubi

Reply:
This is great. I guess I'll review a bunch of other actions where I'm doing that unecessarly. Thanks!